docs/docs/cmd/entra/m365group/m365group-conversation-list.mdx

```diff
@@ -15,20 +15,29 @@ m365 entra m365group conversation list [options]
 ## Options
 
 ```md definition-list
-`-i, --groupId <groupId>`
-: The ID of the Microsoft 365 group
+`-i, --groupId [groupId]`
+: The ID of the Microsoft 365 Group. Specify either `groupId` or `groupName`, but not both.
+
+`-n, --groupName [groupName]`
```

Comment:
TBH I do not feel it. If we in `m365group get/remove/renew/teamify....` use `displayName` why here we add `groupName` instead of `groupDisplayName` seems inconsistent to me and I would propose to always refer to this option as `displayName` and `groupDispalyName`

Reply:
`groupName` is shorter than `groupDisplayName`. It's a bit like we shorten UPN to `userName`.

Reply:
When doing a quick search on these words in our code base, I think `groupName` is the best pick.

* `--groupName`: 30 hits
* `--groupDisplayName`: 6 hits

Reply:
I will keep this without the change.